src/github.com/getlantern/flashlight/pubsub/pubsub.go

```diff
@@ -0,0 +1,24 @@
+// package pubsub provides a global publish and subscribe infrastructure
+// for lantern
+package pubsub
+
+import (
+	"reflect"
+
+	"github.com/asaskevich/EventBus"
+)
+
+var (
+	bus = EventBus.New()
+)
+
+// Pub publishes the given interface to any listeners for that interface.
+func Pub(topic interface{}) {
```

Comment:
ints work for me too.  The only problem is that I'd have preferred to have the constants in the module that defines what they're about (e.g. geolookup in this case).  But it's no big deal either way.

If you want to really enforce that constants are used you can make topics instances of a structure that is local to the pubsub package.